deep-learning/_posts/2017-02-10-vnet.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: review
+title: V-Net:Fully Convolutional Neural Networks for Volumetric Medical Image Segmentation
+tags: deep-learning CNN segmentation medical
+cite:
+  - V-Net
+  - http://campar.in.tum.de/pub/milletari2016Vnet/milletari2016Vnet.pdf
+---
+
+<style>
+  main.page-content p {
```

Comment:
Please replace `main.page-content` by `div.post-content`. It was a mistake (my mistake) to use `main.page-content` (it adds justification outside the text of the review)